core/lib/server/pub-capabilities.db.test.ts

```diff
@@ -0,0 +1,150 @@
+import { describe, expect, it } from "vitest";
+
+import { CoreSchemaType, MemberRole } from "db/public";
+
+import type { Seed } from "~/prisma/seed/seedCommunity";
+import { mockServerCode } from "../__tests__/utils";
+
+await mockServerCode();
+
+const seed = {
+	community: {
+		name: "test-pub-capabilities",
+		slug: "test-pub-capabilities",
+	},
+	users: {
+		admin: {
+			role: MemberRole.admin,
+		},
+		editor: {
+			role: MemberRole.editor,
+		},
+		stage1Editor: {
+			role: MemberRole.contributor,
+		},
+		stage2Editor: {
+			role: MemberRole.contributor,
+		},
+		contributor: {
+			role: MemberRole.contributor,
+		},
+	},
+	pubFields: {
+		Title: { schemaName: CoreSchemaType.String },
+		Description: { schemaName: CoreSchemaType.String },
+		"Some relation": { schemaName: CoreSchemaType.String, relation: true },
+	},
+	pubTypes: {
+		"Basic Pub": {
+			Title: { isTitle: true },
+			"Some relation": { isTitle: false },
+		},
+		"Minimal Pub": {
+			Title: { isTitle: true },
+		},
+	},
+	stages: {
+		"Stage 1": {
+			members: {
+				stage1Editor: MemberRole.editor,
+			},
+		},
+		"Stage 2": {
+			members: {
+				stage2Editor: MemberRole.editor,
+			},
+		},
+	},
+	pubs: [
+		{
+			pubType: "Basic Pub",
+			values: {
+				Title: "Some title",
+			},
+			stage: "Stage 1",
+		},
+		{
+			pubType: "Basic Pub",
+			values: {
+				Title: "Another title",
+			},
+			stage: "Stage 2",
+			relatedPubs: {
+				"Some relation": [
+					{
+						value: "test relation value",
+						pub: {
+							pubType: "Basic Pub",
+							values: {
+								Title: "A pub related to another Pub",
+							},
+						},
+					},
+				],
+			},
```

Comment:
i know you didn't write these tests per se, but i think we should add tests for pub members as well no?

Reply:
oh right I missed that! I can add that

Reply:
https://github.com/pubpub/platform/pull/858

caught a very small bug in the seed script :)